backend/Virkarekisteri/src/Repositories/ChangeLogRepository.cs

```diff
@@ -14,7 +14,7 @@ public class ChangeLogRepository(VirkarekisteriDb db) : IChangeLogRepository
 {
     public async Task<ChangeLog> AddChangeLogEntry(ChangeLog changeLogEntry)
     {
-        changeLogEntry.Timestamp = DateTime.UtcNow;
+        changeLogEntry.Timestamp = DateTime.UtcNow.ToLocalTime();
```

Comment:
What timezone might this "local time" be on the server which happens to run this. Not sure how it works, but I think we should explicitly specify a timezone.

Reply:
Ok, actually I just set a setting in azure which I believe makes the function app's time zone be our time zone.